### PR TITLE
Feat/skeletons

### PR DESCRIPTION
**Pull request by pedrohmorais**
## What this PR does

Creates the `Skeleton` components with some sub-components.

## How to test
Run storybook and go to `Skeleton` story.

### CODE REVIEW

File: components/Skeleton/Skeleton.unit.test.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import Skeleton from './Skeleton';
+
+describe('Skeleton component', () => {
+  it('Should match the snapshot of a simple skeleton', () => {
+    const simpleSkeleton = <Skeleton />;
+    expect(renderer.create(simpleSkeleton).toJSON()).toMatchSnapshot();
+  });
+
+  it('Should match the snapshot of a text skeleton', () => {
+    const simpleSkeleton = <Skeleton.Text />;
```
Comment (renatogalvones) [inline components/Skeleton/Skeleton.unit.test.jsx]: nitpicking: here the vars are all `simpleSkeleton` but only one of them is really `simple`. You can change all of them to `skeleton` ou customize one by one with their respective name like `circleSkeleton`.

File: components/Skeleton/sub-components/SkeletonCircle.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+import SkeletonBase from './SkeletonBase';
+
+const SkeletonCircle = styled(SkeletonBase)`
+  border-radius: 50%;
+`;
+
+SkeletonCircle.displayName = 'SkeletonCircle';
+SkeletonCircle.propTypes = {};
```
Comment (renatogalvones) [inline components/Skeleton/sub-components/SkeletonCircle.jsx]: If props are not being used why having this?

File: components/Skeleton/sub-components/SkeletonRect.jsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+import SkeletonBase from './SkeletonBase';
+
+const SkeletonRect = styled(SkeletonBase)``;
+
+SkeletonRect.displayName = 'SkeletonRect';
+
+SkeletonRect.propTypes = {};
```
Comment (renatogalvones) [inline components/Skeleton/sub-components/SkeletonRect.jsx]: If props are not being used why having this?

File: components/Skeleton/sub-components/SkeletonRect.jsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+SkeletonRect.defaultProps = {};
```
Comment (renatogalvones) [inline components/Skeleton/sub-components/SkeletonRect.jsx]: If props are not being used why having this?

File: components/Skeleton/sub-components/SkeletonTag.jsx [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+  margin-right: 8px;
+
+  ${({
+    size,
+    theme: {
+      baseFontSize,
+      spacing: { xxsmall },
+    },
+  }) => {
+    const paddings = xxsmall * 2;
```
Comment (renatogalvones) [inline components/Skeleton/sub-components/SkeletonTag.jsx]: nitpicking: Why `paddings` in plural?

File: components/Skeleton/sub-components/SkeletonText.jsx [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components';
+import SkeletonBase from './SkeletonBase';
+
+const SkeletonText = styled(SkeletonBase)`
+  border-radius: 4px;
+`;
+
+SkeletonText.displayName = 'SkeletonText';
+
+SkeletonText.propTypes = {};
```
Comment (renatogalvones) [inline components/Skeleton/sub-components/SkeletonText.jsx]: If props are not being used why having this?

File: components/Skeleton/sub-components/SkeletonText.jsx [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+
+SkeletonText.defaultProps = {};
```
Comment (renatogalvones) [inline components/Skeleton/sub-components/SkeletonText.jsx]: If props are not being used why having this?

File: stories/Skeleton/Skeleton.story.jsx [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+          <Col xsmall={2} small={4} medium={6}>
+            {example.component}
+          </Col>
+        </Row>
+      ))}
+    </Container>
+  </Tab>
+);
+
+const description = 'A component that simulate another while it is loading.';
```
Comment (renatogalvones) [inline stories/Skeleton/Skeleton.story.jsx]: This description is not accurate. Maybe: "A component that replaces others while they are loading or not ready to render."

File: components/Skeleton/Skeleton.jsx [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+  height: '14px',
+  size: 'medium',
+  theme: {
+    baseFontSize: defaultBaseFontSize,
+    spacing,
+  },
+};
+
+Skeleton.propTypes = {
+  type: PropTypes.oneOf(['rect', 'circle', 'text', 'Button', 'Tag']),
```
Comment (rapahaeru) [inline components/Skeleton/Skeleton.jsx]: Why `Tag` and `Button` props are in uppercase letter?
Is this on purpose? 
Cause in `type` condition, if the user that implements this component putint `button` instead `Button`, will crash.

**Comment by rapahaeru** [comment]
if the all loaders (circular, progress and skeletons) stays in the same item on storybook manu, called loaders?
WTDY?

**Comment by pedrohmorais** [comment]
@renatogalvones  Updated with master

**Pull request merged**